astroid/brain/brain_builtin_inference.py

```diff
@@ -693,7 +693,7 @@ def infer_issubclass(callnode, context=None):
     return nodes.Const(issubclass_bool)
 
 
-def infer_isinstance(callnode, context=None):
+def infer_isinstance(callnode, context: InferenceContext | None = None):
     """Infer isinstance calls
 
     :param nodes.Call callnode: an isinstance call
```

Comment:
The docstring typing is duplicated and can be removed imo. (There are other instances of this)